wwwroot/css/awstats_bw.css

```diff
@@ -1,31 +1,50 @@
-body { font: 11px verdana, arial, helvetica, sans-serif; background-color: #FFFFFF; margin-top: 0 }
-.aws_bodyl { }
-.aws_border { background-color: #BBBBBB; padding: 1px 1px 1px 1px; margin-top: 0 }
-.aws_title  { font: 13px verdana, arial, helvetica, sans-serif; font-weight: bold; background-color: #BBBBBB; text-align: center; margin-bottom: 0; padding: 1px 1px 1px 1px; }
-.aws_blank { font: 13px verdana, arial, helvetica, sans-serif; background-color: #FFFFFF; text-align: center; margin-bottom: 0; padding: 1px 1px 1px 1px; }
+body { body { font: 0.75rem sans-serif, system-ui; background-color: #FFFFFF; margin-top: 0; margin-bottom: 0; } }
+.aws_bodyl { background-image: url(/awstatsicons/other/backleft.png); background-repeat: repeat-y; }
```

Comment:
May be we should use $DirIcons here ?

Reply:
Hi @eldy
It's a copypaste from awstats_default. I don't know the difference between awstats_default.css and awstats_bw.css, and their use except for example, because css is hardcoded in awstats.pl. Maybe we can keep only one.
By the way, did you hardcoded the css for a special purpose ? Or maybe we can include a default "$StyleSheet" from css folder and remove the hardcoded one ?

For the backleft image, maybe we just could remove this image. And the button.gif also.

Another consider i have is to refactor all the calculations for tables. For example, in HTMLMainMonthly, these are written 3 times, and calculated 2 times (or more if there are multiples plugins). We could just use the ShowGraph plugin calculated vars and then dispatch them for graph plugins and rendering bars table and data table.